pkg/utils/utils.go

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2021 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
```

Comment:
We could probably move those functions to `pkg/cli` or `pkg/cmd` 👼🏼 We try to avoid as much as possible package named `utils` or `common`, … 😅

Reply:
sure thing

Reply:
Moved to `pkg/cli`